crates/sui-prover/tests/snapshots/reals.fail.move.snap

```diff
@@ -11,25 +10,3 @@ error: prover::ensures assertion does not hold
    │   ^^^^^^^^^^^^^^^^^^^^^^^^^^
    │
    =     at tests/inputs/reals.fail.move:16: foo_spec
-   =         x = 0u64
-   =     at sui-framework/packages/move-stdlib/sources/u64.move:90
-   =         x = 0u64
-   =     at tests/inputs/reals.fail.move:7: foo
-   =         result = 1u64
-   =     at tests/inputs/reals.fail.move:8: foo
-   =         res#1#0 = 1u64
-   =     at tests/inputs/reals.fail.move:19: foo_spec
-   =         x = 1u64
-   =     at sui-framework/packages/move-stdlib/sources/real.move:28: from_u64
-   =         result = 1.0
-   =     at sui-framework/packages/move-stdlib/sources/real.move:29: from_u64
-   =         x = 7u64
-   =     at sui-framework/packages/move-stdlib/sources/real.move:28: from_u64
-   =         result = 7.0
-   =     at sui-framework/packages/move-stdlib/sources/real.move:29: from_u64
-   =         the_real = 1.0/7.0
-   =     at tests/inputs/reals.fail.move:12: show_real
-   =         result = 1.0/7.0
-   =     at tests/inputs/reals.fail.move:13: show_real
-   =         y_real#1#0 = 1.0/7.0
```

Comment:
let's add `log::var(y_real)` to the test to check the pretty-printing of `1.0/7.0`